src/projectscene/qml/Audacity/ProjectScene/trackspanel/RealtimeEffectListItem.qml

```diff
@@ -0,0 +1,134 @@
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+
+import Muse.Ui
+import Muse.UiComponents
+
+ListItemBlank {
+    id: root
+
+    property var item: null
+    property var availableEffects: null
+    property var handleMenuItemWithState: null
+
+    height: 24
+    clip: false // should be true?
+    background.color: "transparent"
+    hoverHitColor: "transparent"
+
+    RowLayout {
+        anchors.fill: parent
+        spacing: 4
+
+        FlatButton {
+            id: gripButton
+
+            Layout.preferredWidth: 14
+            Layout.preferredHeight: 16
+            Layout.alignment: Qt.AlignVCenter
+            backgroundRadius: 0
+
+            visible: true
+            transparent: true
+            hoverHitColor: ui.theme.buttonColor
+            mouseArea.cursorShape: Qt.SizeAllCursor
+
+            // do not accept buttons as FlatButton's mouseArea will override
+            // DockTitleBar mouseArea and effect will not be draggable
+            mouseArea.acceptedButtons: Qt.NoButton
+
+            contentItem: StyledIconLabel {
+                iconCode: IconCode.TOOLBAR_GRIP
+            }
+
+        }
+
+        FlatButton {
+            id: powerButton
+
+            // TODO: make that from TrackEffectsSection reusable
+            Layout.margins: 0
+            Layout.alignment: Qt.AlignVCenter
+            Layout.preferredWidth: root.height
+            Layout.minimumHeight: root.height
+            Layout.maximumHeight: root.height
+            height: width
+
+            icon: IconCode.BYPASS
+            iconFont: ui.theme.toolbarIconsFont
+
+            accentButton: true
+
+            onClicked: {
+                accentButton = !accentButton
+            }
+        }
+
+        // Wrappinng a FlatButton in a Rectangle because of two bad reasons:
+        // * I don't find a `border` property for `FlatButton`
```

Comment:
If you need a border for FlatButton you should override backgroundItem

Reply:
Thanks for the suggestion, I looked into this.
The default for FlatButton is that Component of ID `defaultBackgroundComponent`. It does everything I want my button to do, except for the border thing (it uses `ui.theme.borderWidth`, which equals 0, though). I haven't found a readily solution whereby I can reuse it without duplicating it or changing the framework to expose that property in the FlatButton. That'd be the right way, but is maybe there's another correct way ?

Reply:
I will suggest duplicating, overriding backgroundItem is done exactly for such cases, so as not to create a lot of properties

Reply:
Yeah, but then I have a button whose only difference to the rest of the UI is that it has a non-zero border. If I duplicate it and `MenuButton` changes on the framework, my button will not get automatically updated.
So there seems not to be a way I overlooked. Will go for the duplication now but will look into making a PR in the framework.

Reply:
I just gave it a try: it's not enough to just duplicate `defaultBackgroundComponent`, I'd have to duplicate the entire `FlatButton`.
So nevermind, for now, it'll be the rectangle wrapper.

>  overriding backgroundItem is done exactly for such cases, so as not to create a lot of properties

And how about
FlatButton.qml:
```qml
FocusScope {
    property Component backgroundItem: defaultBackgroundComponent
    property var backgroundComponentLoaderItem: backgroundLoader.item // pity backgroundItem was not called `backgroundComponent` ...
    
    Loader {
        id: backgroundLoader
        sourceComponent: root.backgroundItem
    }
```
and then
```qml
  FlatButton {
      Component.onCompleted: {
          backgroundComponentLoaderItem.border.width = 1
      }
  }
```
This only exposes one more property and gives the client control over the properties of the default item. (Only a pity that I have to use `Component.onCompleted`)

Reply:
Like I said,
> I just gave it a try: it's not enough to just duplicate defaultBackgroundComponent, I'd have to duplicate the entire FlatButton.

Reply:
show how you tried to duplicate

Reply:
I threw the experiment away. You can see for yourself that defaultBackgroundComponent has dependencies on FlatButton.